.github/workflows/emun.yml

```
@@ -0,0 +1,16 @@
+name: EMUN Deployment
+on:
```

YAMLlint (1.38.0)

[warning] truthy value should be one of [false, true]

(truthy)